tests/functional/vault/test_losses.py

```diff
@@ -14,29 +14,33 @@ def strategy(gov, vault, TestStrategy):
     yield gov.deploy(TestStrategy, vault)
 
 
-def test_reporting(vault, strategy, gov, token, rando):
+def test_losses(chain, vault, strategy, gov, token):
     vault.addStrategy(strategy, 1000, 1000, 0, {"from": gov})
     token.approve(vault, 500, {"from": gov})
     vault.deposit(500, {"from": gov})
 
+    chain.sleep(8640)
```

Comment:
as above, maybe introduce a var to name the time frame?

Reply:
`DAY // 10`